### ISSUE TITLE
Telegram MAC crashes on rollover at smiley dropdown

### ISSUE THREAD

**Issue by hirasso** [overtake/telegram#428]
### Steps to reproduce
1. open telegram
2. go to chat
3. try to insert an icon from the select

### Expected behaviour
I would like to select a smiley

### Actual behaviour
Telegram crashes

### Configuration
MAC OSX 10.12.4

**Version of Telegram Desktop:**
2.92

**Used theme**:
Default

**Comment by overtake** [comment]
@hirasso Hi, are you using version from site?

**Comment by hirasso** [comment]
@overtake Yes, I think so

**Comment by overtake** [comment]
@hirasso I think you doesn't have sticker sets and have gifs?

**Comment by hirasso** [comment]
@overtake only Smileys and GIFs now...

**Comment by overtake** [comment]
@hirasso yes. sorry for this. i was fixed its on appstore version. You can install one of sticker set and its will be deciced. Or Wait next update

**Comment by hirasso** [comment]
No problem! Call me old fashioned, but I'm not using the stickers anyway ;)
